examples/applications/gateway-authentication/gateways.yaml

```diff
@@ -21,20 +21,20 @@ gateways:
     topic: input-topic
     parameters:
       - sessionId
-    produceOptions:
+    produce-options:
```

Comment:
this reads like a breaking change and we have stuff in production, please don't do it

Reply:
no it's not, it's an alias and they both work